src/comps/context/ThemeProvider.js

```diff
@@ -1,58 +1,128 @@
 import React from "react";
 
-import { createMuiTheme, ThemeProvider as Provider } from "@material-ui/core";
+import { createTheme, ThemeProvider as Provider, useMediaQuery } from "@mui/material";
 
-const theme = createMuiTheme({
-	palette: {
-		primary: {
-			main: "#e66767",
-			contrastText: "#fff"
-		},
-		secondary: {
-			// light: "#ff7961",
-			main: "#546de5",
-			// dark: "#ba000d",
-			contrastText: "#fff"
-		}
-	},
-	typography: {
-		fontFamily: `'Roboto Condensed', sans-serif`,
-		htmlFontSize: 16,
-		h1: {
-			fontSize: "3rem",
-			fontWeight: 700,
-			margin: "0.5rem 0"
-		},
-		h2: {
-			fontSize: "2.6rem",
-			fontWeight: 700,
-			margin: "0.5rem 0"
-		},
-		h3: {
-			fontSize: "2.2rem",
-			fontWeight: 500,
-			margin: "0.5rem 0"
-		},
-		h4: {
-			fontSize: "1.8rem",
-			fontWeight: 500,
-			margin: "0.5rem 0"
-		},
-		h5: {
-			fontSize: "1.4rem",
-			fontWeight: 400,
-			margin: "0.5rem 0"
-		},
-		h6: {
-			fontSize: "1rem",
-			fontWeight: 400,
-			margin: "0.5rem 0"
-		}
-	}
+const ThemeContext = React.createContext({
+	toggleColorMode: () => {},
+	colorMode: false
 });
 
 const ThemeProvider = props => {
-	return <Provider theme={theme}>{props.children}</Provider>;
+	const browserPreference = useMediaQuery("(prefers-color-scheme: dark)");
+	const [storedPreference, setStoredPreference] = React.useState(window.localStorage.getItem("mode"));
+
+	const prefersDarkMode = storedPreference !== null ? storedPreference === "dark" : browserPreference;
+	const toggle = React.useMemo(
+		() => () =>
+			setStoredPreference(() => {
+				const output = prefersDarkMode ? "light" : "dark";
+				window.localStorage.setItem("mode", output); // persist in localStorage
+				return output;
+			}),
+		[prefersDarkMode]
+	);
+
+	const theme = React.useMemo(
+		() =>
+			createTheme({
+				palette: {
+					mode: prefersDarkMode ? "dark" : "light",
+					primary: {
+						main: "#e66767",
+						contrastText: "#fff"
+					},
+					secondary: {
+						main: "#546de5",
+						contrastText: "#fff"
+					},
+					button: {
+						main: prefersDarkMode ? "rgba(255, 255, 255, 0.87)" : "rgba(0, 0, 0, 0.87)"
```

Comment:
Something here seems to be a bit off since buttons are displayed as if they were in dark mode even in light mode

Also the calendar (and text) seem slightly bigger than before but I can't pinpoint why

![ThingAN2213](https://user-images.githubusercontent.com/90653148/192077574-32a9b06a-365a-432d-a55d-c3c71f0b79f6.png)

Reply:
The fullcalendar components use `em` for styling (baked into the library), which indirectly reference the `<body>`'s `font-size: 0.875rem` specification. The updated MUI theme changes the `<body>` font size to 1rem at `htmlFontSize: 16`, rather than `0.875rem` as it was previously. To fix it, a parent div has been added that gives `<Calendar>` the expected parent fontSize.